### PR TITLE
Further manual editing

### PR DESCRIPTION
**Pull request by prckent**
More edits from Charlie Horak. Verified manual still builds.

### CODE REVIEW

File: manual/lab_advanced_molecules.tex [hunk 1]
```diff
@@ -124,29 +124,29 @@ \subsection{Generation of a Hatree-Fock wave-function with GAMESS}
...
-defined before the wave-function). The name of the particle set and wave-function files should now be h2o.ptcl.xml
-and h2o.wfs.xml, respectively. Study both files and submit when you are ready. Notice that the
-location of the ECPs has been set for you, in your own calculations you have to make
+defined before the wavefunction). The name of the particle set and wavefunction files should now be \texttt{h2o.ptcl.xml}
+and \texttt{h2o.wfs.xml}, respectively. Study both files and submit when you are ready. Notice that the
+location of the ECPs has been set for you; in your own calculations you have to make
 sure you obtain the ECPs from the appropriate libraries and convert them to QMCPACK
-format using ppconvert. This is a good time to study section C, which contains a review
-of the main parameters in the optimization XML block, while this calculation finishes. The
+format using ppconvert. While these calculations finish is a good time to study Section 19.8, ``Appendix C: Wavefunction optimization XML block," which contains a review
```
Comment (ye-luo) [inline manual/lab_advanced_molecules.tex]: Not a `\ref`?

File: manual/examples.tex [hunk 2]
```diff
@@ -77,4 +77,4 @@ \section{Using Nexus}
 %\subsection{LiH crystal with Quantum ESPRESSO orbitals}
 %The input files are in the directory \ishell{nexus/examples/qmcpack/LiH}.
 %
-%The \ishell{LiH.py} Nexus script uses CASINO-formatted Trail-Needs pseudopotentials (see Section~\ref{subsec:CASINO}) for Li and H in a subdirectory named \ishell{pseudopotentials} (both UPF and CASINO  formats, named \ishell{Li.TN-DF.upf}, \ishell{Li.pp.data}, \ishell{H.TN-DF.upf}, and \ishell{H.pp.data}, respectively), generates the orbitals using Quantum ESPRESSO, then runs QMCPACK to optimize the Jastrow and run DMC for LiH with periodic boundary conditions.
+%The \ishell{LiH.py} Nexus script uses CASINO-formatted Trail-Needs pseudopotentials (see Section \ref{subsec:CASINO}) for Li and H in a subdirectory named \ishell{pseudopotentials} (both UPF and CASINO  formats, named \ishell{Li.TN-DF.upf}, \ishell{Li.pp.data}, \ishell{H.TN-DF.upf}, and \ishell{H.pp.data}, respectively), generates the orbitals using Quantum ESPRESSO, then runs QMCPACK to optimize the Jastrow and run DMC for LiH with periodic boundary conditions.
```
Comment (ye-luo) [inline manual/examples.tex]: Seem like an unexpected change

File: manual/lab_advanced_molecules.tex (hunk 1 above)
Comment (prckent) [reply manual/lab_advanced_molecules.tex]: Fixed. Good catch.

File: manual/examples.tex (hunk 2 above)
Comment (prckent) [reply manual/examples.tex]: Fixed, but this is commented out.

**Comment by prckent** [comment]
Wait for next update

**Comment by prckent** [comment]
Fixed broken LaTeX. OK to merge.

**Pull request merged**